### PR TITLE
Efrs 820 test-model page redesign 

### PR DESCRIPTION
**Pull request by YarasKo**

### CODE REVIEW

File: ui/src/app/features/app-user-list/application-user-list-facade.ts [hunk 1]
```diff
@@ -83,8 +91,10 @@ export class ApplicationUserListFacade implements IFacade {
 
     this.availableRoles$ = combineLatest([allRoles$, this.userRole$, this.organizationRole$]).pipe(
       map(([allRoles, userRole, organizationRole]) => {
-        if (organizationRole !== 'USER') {
+        if (organizationRole === Role.OWNER) {
           return allRoles;
+        } else if (organizationRole === Role.ADMINISTRATOR) {
+          return allRoles.filter(role => role !== Role.OWNER);
```
Comment (olehkhomyk) [inline ui/src/app/features/app-user-list/application-user-list-facade.ts:97]: Remove this, and make this method like on develop branch

File: ui/src/app/features/app-user-list/application-user-list-facade.ts [hunk 2]
```diff
@@ -74,7 +74,15 @@ export class ApplicationUserListFacade implements IFacade {
...
-        return organizationRole !== Role.USER ? organizationRole : applicationRole;
+        if (organizationRole !== Role.USER) {
+          if (organizationRole === Role.OWNER) {
+            return organizationRole;
+          }
+
+          if (organizationRole === Role.ADMINISTRATOR) {
+            return applicationRole === Role.OWNER ? applicationRole : organizationRole;
+          }
+        }
```
Comment (olehkhomyk) [inline ui/src/app/features/app-user-list/application-user-list-facade.ts:85]: Remove this, and make this method like on develop branch

File: ui/src/app/features/face-recognition/recognition-result/recognition-result.component.html [hunk 3]
```diff
@@ -1,36 +1,48 @@
...
-                    {{requestInfo.response}}
+    <mat-expansion-panel>
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          {{'test_model.response' | translate}}
+        </mat-panel-title>
+      </mat-expansion-panel-header>
+      <textarea class="result-textarea" id="result-textarea-response" rows="16" disabled>
+                    {{ requestInfo.response }}
                 </textarea>
```
Comment (olehkhomyk) [inline ui/src/app/features/face-recognition/recognition-result/recognition-result.component.html:44]: fix tabs

File: ui/src/app/features/face-recognition/recognition-result/recognition-result.component.html [hunk 4]
```diff
@@ -1,36 +1,48 @@
...
-            <div class="separate-line"></div>
+    <mat-expansion-panel>
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          {{'test_model.request' | translate}}
+        </mat-panel-title>
+      </mat-expansion-panel-header>
+      <textarea class="result-textarea" id="result-textarea-request" rows="10" disabled>
+                    {{ requestInfo.request }}
+                </textarea>
```
Comment (olehkhomyk) [inline ui/src/app/features/face-recognition/recognition-result/recognition-result.component.html:33]: fix tabs

File: ui/src/app/features/user-table/user-table.component.ts [hunk 5]
```diff
@@ -61,9 +61,16 @@ export class UserTableComponent extends TableComponent implements OnInit, OnChan
...
       && this.userRole !== Role.USER
+      && user.role !== Role.OWNER
       && this.availableRoles.indexOf(user.role) > -1;
   }
 
+  isDisabled(element): any {
+    return (element.userId === this.currentUserId && element.role !== Role.OWNER) ||
+      (this.userRole === Role.ADMINISTRATOR && element.role !== Role.OWNER) ||
+      (this.userRole === Role.OWNER && element.role !== Role.OWNER);
+  }
```
Comment (olehkhomyk) [inline ui/src/app/features/user-table/user-table.component.ts:72]: Remove this, and make this method like on develop branch

File: ui/src/app/features/user-table/user-table.component.ts [hunk 6]
```diff
@@ -61,9 +61,16 @@ export class UserTableComponent extends TableComponent implements OnInit, OnChan
   isRoleChangeAllowed(user: AppUser): boolean {
     return user.userId !== this.currentUserId
       && this.userRole !== Role.USER
+      && user.role !== Role.OWNER
```
Comment (olehkhomyk) [inline ui/src/app/features/user-table/user-table.component.ts:64]: remove this expression

**Review by isobolev — APPROVED** [review]

**Pull request merged**